dao/Dao/Vote.hs

```diff
@@ -185,14 +165,9 @@ data VoteDynamicConfigDatum = VoteDynamicConfigDatum
   , vdcFungibleVotePercent :: BuiltinData
   }
 
-unstableMakeIsData ''VoteMinterAddress
-unstableMakeIsData ''VoteMinterTxOut
-unstableMakeIsData ''VoteMinterTxInInfo
-unstableMakeIsData ''VoteMinterScriptPurpose
-unstableMakeIsData ''VoteMinterScriptContext
-unstableMakeIsData ''VoteMinterTxInfo
 unstableMakeIsData ''VoteMinterDynamicConfigDatum
 
+-- | Redeemer for 'Triphut.Vote.Script.validateVote' validator
```

Comment:
This should be `Triphut.Vote.Script.validateVote`

Reply:
Sorry @Randy1Burrell - missed your comment here. Just fixed that now.